### PR TITLE
Update HasTranslations.php

### PR DESCRIPTION
**Pull request by altynbek07**
To fix this issues:
https://github.com/spatie/laravel-translatable/issues/195
https://github.com/spatie/laravel-translatable/pull/203

Now if value is empty return value from fallback locale.

### CODE REVIEW

**Comment by altynbek07** [comment]
@Omranic Hi. What do you think about it?

**Comment by Omranic** [comment]
@altynbek07 Can you provide a use case where I can see this in action?
Also what if there's no translation for the fallback locale, shouldn't we still return null as a third clause?

**Comment by altynbek07** [comment]
> @altynbek07 Can you provide a use case where I case see this in action?
> Also what if there's no translation for the fallback locale, shouldn't we still return null as a third clause?


For example. I have three locales: `en`, `ru` and `kk`. My main and fallback locale is `en`. 
I have these data in my database:
In some table (for example `posts`), column `title` (column type is `JSON` or `TEXT` doesn't matter):
First case:
```json
{
 "en": "Some title",
 "ru": "",
 "kk": ""
}
```
Second case:
```json
{
 "en": "Some title"
}
```
In first case, if I change my language to `ru`, I will get an empty string.
```php
$post->title // ''
```
In second case, if I change my language to `ru`, I will get 'Some title'.
```php
$post->title // 'Some title'
```

---
My fix just looks at an empty string, and returns a value from the fallback locale.

File: src/Traits/HasTranslations.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function getTranslations($key): array
     public function attributesToArray()
     {
         $values = array_map(function ($attribute) {
-            return $this->getTranslation($attribute, config('app.locale')) ?: null;
+            return $this->getTranslation($attribute, config('app.locale')) ?: $this->getTranslation($attribute, config('translatable.fallback_locale'));
```
Comment (altynbek07) [inline src/Traits/HasTranslations.php:68]: Use `config('app.fallback_locale')` instead of `config('translatable.fallback_locale')`

**Comment by altynbek07** [comment]
@Omranic Use `config('app.fallback_locale')` instead of `config('translatable.fallback_locale')`. I can't edit this file for some reason.

**Comment by Omranic** [comment]
If I understand correctly, this is already supported by the core package, see here https://github.com/spatie/laravel-translatable/blob/master/src/HasTranslations.php#L196-L202

If you believe that's something different, feel free to reopen this issue.

**Pull request closed without merging**